libraries/provider_hab_package.rb

```diff
@@ -114,8 +114,6 @@ def platform_target
             'target=x86_64-windows'
           elsif platform_family?('rhel') && node['platform_version'].to_f < 6.0
             'target=x86_64-linux-kernel2'
-          elsif platform_family?('centos') && node['platform_version'].to_f < 6.0
```

Comment:
@sirajrauff why were these lines removed? They seem unrelated to the rest of the changes in this PR.

Reply:
This came up in the latest Cookstyle, 'centos' isn't a platform family.

Reply:
Ah that totally makes sense, thanks :D